pkg/util/limitrange/limitrange_test.go

```diff
@@ -160,79 +160,194 @@ func TestSummarize(t *testing.T) {
 }
 
 func TestTotalRequest(t *testing.T) {
-	containers := []corev1.Container{
-		{
-			Resources: corev1.ResourceRequirements{
-				Requests: corev1.ResourceList{
-					corev1.ResourceCPU:    resource.MustParse("1"),
-					corev1.ResourceMemory: resource.MustParse("2Gi"),
-				},
-			},
-		},
-		{
-			Resources: corev1.ResourceRequirements{
-				Requests: corev1.ResourceList{
-					corev1.ResourceCPU: resource.MustParse("1500m"),
-					"example.com/gpu":  resource.MustParse("2"),
-				},
-			},
-		},
-		{
-			Resources: corev1.ResourceRequirements{
-				Requests: corev1.ResourceList{
-					corev1.ResourceCPU:    resource.MustParse("4"),
-					corev1.ResourceMemory: resource.MustParse("2Gi"),
-				},
-			},
-		},
-		{
-			Resources: corev1.ResourceRequirements{
-				Requests: corev1.ResourceList{
-					corev1.ResourceCPU: resource.MustParse("1500m"),
-					"example.com/gpu":  resource.MustParse("2"),
-				},
-			},
-		},
-	}
 	cases := map[string]struct {
 		podSpec *corev1.PodSpec
 		want    corev1.ResourceList
 	}{
-		"sum up main containers": {
+		"pod without init containers": {
 			podSpec: &corev1.PodSpec{
-				Containers: containers[:2],
+				Containers: []corev1.Container{
+					*testingutil.MakeContainer().
+						WithResourceReq(corev1.ResourceCPU, "1").
+						WithResourceReq(corev1.ResourceMemory, "2Gi").
+						Obj(),
+					*testingutil.MakeContainer().
+						WithResourceReq(corev1.ResourceCPU, "1500m").
+						WithResourceReq("example.com/gpu", "2").
+						Obj(),
+				},
 			},
 			want: corev1.ResourceList{
 				corev1.ResourceCPU:    resource.MustParse("2500m"),
 				corev1.ResourceMemory: resource.MustParse("2Gi"),
 				"example.com/gpu":     resource.MustParse("2"),
 			},
 		},
-		"one init wants more": {
+		"pod only with regular init containers": {
```

Comment:
nit: "pod with init containers"

Reply:
nit: consider a description of not only the setup, but what we hope to verify with a particularly test

e.g "pod with init containers. request max of all init containers"

consider these sorts of descriptions for other tests too.

Reply:
> nit: "pod with init containers"

I think it's best to leave `regular init containers` since we aren't passing sidecar containers which might be considered InitContainers as well.

Reply:
> nit: consider a description of not only the setup, but what we hope to verify with a particularly test
> 
> e.g "pod with init containers. request max of all init containers"
> 
> consider these sorts of descriptions for other tests too.

Done.